assets/js/googlesitekit/components-gm3/Checkbox.js

```diff
@@ -58,6 +58,7 @@ export default function Checkbox( {
 	onKeyDown,
 	loading,
 	alignLeft,
+	label,
```

Comment:
Just a reminder to apply the same update here as to the GM2 Checkbox and to update the GM3 stories too.

We also have test coverage for the GM3 Checkbox and it would be good to add an additional test to cover the new prop.